src/MSBuild/MSBuild.csproj

```diff
@@ -216,16 +216,9 @@
     <Content Include="MSBuild.rsp" Pack="false">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <None Include="MSBuild\Microsoft.Build.CommonTypes.xsd">
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="MSBuild\Microsoft.Build.Core.xsd">
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="Microsoft.Build.xsd">
-      <SubType>Designer</SubType>
+    <XsdsForVS Include="**\*.xsd" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
```

Comment:
There are two "Default exclude" items here,  so presumably "DefaultItemExcludes" doesn't include the "DefaultExcludesInProjectFolder". How does one know or find out how many default exclude items there are?

Reply:
There's probably a better way, but I build a project that excludes those, catch it in a debugger, break on the code that handles that in `Project.cs`, and inspect what it gets turned into.

Reply:
I'm following the SDK's approaches here:

https://github.com/dotnet/sdk/blob/e053f40806bf1e50b01271cbddc57759d24f0b0b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L22-L42

https://github.com/dotnet/sdk/blob/062d9a036ea335833e472d3b3c17174ba62c5c19/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props#L37

No real way to know other than looking for analogies there that I know of.